tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/GraphOperation.java

```diff
@@ -170,6 +184,155 @@ Tensor tensor(int outputIdx) {
     throw new IllegalStateException("Graph tensors must be fetched by running a session");
   }
 
+  /**
+   * Get the number of inputs to the op, not including control inputs.
+   */
+  public int numInputs() {
+    try (PointerScope scope = new PointerScope()) {
+      return TF_OperationNumInputs(getUnsafeNativeHandle());
+    }
```

Comment:
We don't need a pointer scope here since we call straight a native endpoint without allocating anything